### PR TITLE
DOCSP-43808 document non idempotent inserts

### PR DESCRIPTION
**Pull request by mmaville-mdb**
## DESCRIPTION
Documenting non-impodency behavior for sync jobs.

## STAGING
https://deploy-preview-173--docs-relational-migrator.netlify.app/jobs/sync-jobs/#behavior

## JIRA
https://jira.mongodb.org/browse/DOCSP-43808

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/jobs/sync-jobs.txt [hunk 1]
```diff
@@ -40,6 +40,19 @@ Recoverability
...
 .. include:: /includes/fact-cdc-recoverable.rst
 
+Behavior
+--------
+
+Sync jobs are non-impodent by default. Relational Migrator inserts new 
+documents and embeded arrays each time a job is run, with the option to 
+delete each existing collection at the start of the sync dialogue.
+
+You can enable impodent inserts for documents and embedded arrays to
```
Comment (jason-price-mongodb) [inline source/jobs/sync-jobs.txt:50]: Consider rewrite in form: "To XXX, set YYY."

File: source/jobs/sync-jobs.txt [hunk 2]
```diff
@@ -40,6 +40,19 @@ Recoverability
 
 .. include:: /includes/fact-cdc-recoverable.rst
 
+Behavior
+--------
+
+Sync jobs are non-impodent by default. Relational Migrator inserts new
```
Comment (jason-price-mongodb) [inline source/jobs/sync-jobs.txt:46]: Spelling mistake "impodent"

**Review by jason-price-mongodb — CHANGES_REQUESTED** [review]
@mmaville-mdb requires typo fix, and rewrite of statement. Cheers.

File: source/jobs/sync-jobs.txt (hunk 1 above)
Comment (mmaville-mdb) [reply source/jobs/sync-jobs.txt:50]: Good suggestion! I also simplified the sentence and moved a couple of things around, while providing the same info.

File: source/jobs/sync-jobs.txt (hunk 2 above)
Comment (mmaville-mdb) [reply source/jobs/sync-jobs.txt]: Thanks, looks like I had blinders on after swapping out the term accidentally.

**Review by alwaysbemark — APPROVED** [review]
This looks good. We will add this as a UI toggle in the next release or two but this PR correctly reflects the current state of things.

Thanks!

**Pull request merged**